nautobot_device_lifecycle_mgmt/forms.py

```diff
@@ -216,6 +217,85 @@ def clean(self):
             self.add_error(None, msg)
 
 
+class ValidatedSoftwareLCMBulkEditForm(NautobotBulkEditForm):
+    """Validated Software Lifecycle bulk edit form."""
+
+    model = ValidatedSoftwareLCM
+
+    pk = forms.ModelMultipleChoiceField(
+        queryset=ValidatedSoftwareLCM.objects.all(),
+        widget=forms.MultipleHiddenInput,
+    )
+
+    software = DynamicModelChoiceField(
+        queryset=SoftwareVersion.objects.all(),
+        required=False,
+        label="Software Version",
+    )
+    old_software = DynamicModelChoiceField(
```

Comment:
The `old_` fields shouldn't be exposed through the UI. They are only there to keep the old data in the database in case a migration fails and exposing them through the UI is just adding unnecessary confusion.